pkg/server/config/config.go

```diff
@@ -65,12 +67,19 @@ func LoadConfig(cm *core_v1.ConfigMap) error {
 		return fmt.Errorf("validate config failed")
 	}
 
+	modifier(&Config)
+
 	log.Infof("cyclone server config: %v", Config)
 	return nil
 }
 
 // validate validates some required configurations.
 func validate(config *CycloneServerConfig) bool {
+	return true
```

Comment:
Always return true?

Reply:
There is nothing we need to validate by now, I think I need to delete this function at this time.

Reply:
ok